osbenchmark/worker_coordinator/runner.py

```diff
@@ -2696,3 +2697,17 @@ async def __call__(self, opensearch, params):
 
     def __repr__(self, *args, **kwargs):
         return "deploy-ml-model"
+
+class EnableConcurrentSegmentSearch(Runner):
```

Comment:
nit: What do you think about naming as UpdateConcurrentSegmentSearchClusterSettings?

Reply:
ack, makes sense if we're adding the `slice count` param